CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.1)
+project(MyApplication)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${PROJECT_SOURCE_DIR}/modules/")
```

Comment:
Ah damn, that reminds me I forgot to update the bootstrap repos to do

```cmake
set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/modules/" ${CMAKE_MODULE_PATH})
```

instead.

Reply:
Fixed locally, will push soon.